### PR TITLE
[#IOPID-94] Add first LC openapi

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes

Added `api_lollipop_first_consumer.yaml`

#### Motivation and Context

Expose first LC endpoint

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: api_lollipop_first_consumer.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        "400":
+          description: Bad request
+          schema:
+            $ref: '#/definitions/ProblemJson'
+        "401":
+          description: Token null or invalid.
+        "500":
+          description: There was an error
+          schema:
+            $ref: "#/definitions/ProblemJson"
```
[the remark this answers is not in the document]
Comment (gquadrati) [reply api_lollipop_first_consumer.yaml:69]: 401 does not return a ProblemJson, see https://github.com/pagopa/io-backend/blob/0ea5db0e7b776cc26474fe66d851e0e30db912ec/api_backend.yaml#L42

[the remark this answers is not in the document]
Comment (shadowsheep1) [reply api_lollipop_first_consumer.yaml:69]: Yep, sorry. I explain better what I mean. With the provided hint I get the response decoder correctly as I expect it to be with the response types provided above. It's correct to have a "`401`" without response body 👍

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Review by BurnedMarshal — APPROVED** [review]
LGTM. Is correct that all the openapi specification on root folder are not validated from openapi linter? step `validate_api_specification`

**Comment by gquadrati** [comment]
> LGTM. Is correct that all the openapi specification on root folder are not validated from openapi linter? step `validate_api_specification`

I see that too.. I think we should remove the one in root folder and move to openapi folder, but we need to synchronize the change with the app.

**Pull request merged**